bot.py

```diff
@@ -338,6 +338,86 @@ async def display_current_list_of_summoners(ctx):
         await ctx.send(embed=create_embed(embed_data))
 
 
+@bot.command(name="teams", help="Display two teams")
+async def display_teams(ctx):
+    """Make and display teams to bot from list of summoners in json"""
+    try:
+        # typing indicator
+        async with ctx.typing():
+            await asyncio.sleep(1)
+
+        # server id
+        server_id = str(ctx.guild.id)
+
+        file_data = ""
+
+        if os.path.getsize(json_path) == 0 or not os.path.exists(json_path):
+            raise Exception
+
+        # get data from the json file and save to file data
+
+        with open(json_path, "r") as file:
+            file_data = json.load(file)
+
+        if not file_data[server_id] or len(file_data[server_id]) != 10:
```

Comment:
Have you tested if `not file_data[server_id]` work? 
I've used [server_id in file_data.keys()](https://github.com/dhmoon91/discord/pull/32/files#diff-36f6200cd8353111badb874140eba3af6055f4cc5839fbdeb9ede992620828c1R459) to check if `server_id` exists in data.json

Reply:
Oh I think the way you did it makes it easier to read and follow, I'll change that.